src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -1209,7 +1209,8 @@
                     References="@(Reference)"
                     FrameworkReferences="@(FrameworkReference)"
                     Files="@(_packageFile)"
-                    Serviceable="$(Serviceable)"/>
+                    Serviceable="$(Serviceable)"
+                    PackageTypes="$(PackageType)"/>
```

Comment:
The actual NuGet `PackTask` does this (instead of `@(PackageType)`) as well: https://github.com/NuGet/NuGet.Client/blob/50af5271b98ac5cb2896a707569bc4cd1e87a017/src/NuGet.Core/NuGet.Build.Tasks.Pack/NuGet.Build.Tasks.Pack.targets#L224